### PR TITLE
Adds the 1.9.5 fluent-bit rock

### PR DESCRIPTION
**Pull request by claudiubelu**
The ``-DFLB_TRACE=Off`` cmake flag is set in v1.9.5, and the ``-DFLB_CHUNK_TRACE=On`` is set in v2.1.6.

Updated the sanity tests to also test v1.9.5 image.

Updated integration test to pick the v2.1.6 image.

### CODE REVIEW

**Comment by claudiubelu** [comment]
Should be merged after: https://github.com/canonical/fluent-bit-rock/pull/4

**Review by bschimke95 — APPROVED** [review]
LGTM, let's merge after the others PRs are in and please remove the now obsolete commits

**Comment by bschimke95** [comment]
Some conflicts now because of the #4 merge. Please resolve, after that we can merge this as well.

File: tests/sanity/test_rock.py
Comment (bschimke95) [inline tests/sanity/test_rock.py]: nit: we can just parametrize the test instead of having dedicated functions
https://github.com/canonical/cert-manager-rock/pull/8/files#diff-aafa91523686591db7f607be14eddefac92bd43e12c1b4e077de779fd2722e33R27

Comment (claudiubelu) [reply tests/sanity/test_rock.py]: Good point, done.

**Pull request merged**
